src/SamplesApp/UITests.Shared/Windows_UI_ViewManagement/IsScreenCaptureEnabledTests.xaml

```diff
@@ -0,0 +1,16 @@
+﻿<UserControl
+    x:Class="UITests.Shared.Windows_UI_ViewManagement.IsScreenCaptureEnabledTests"
```

Comment:
Indentation for XAML files ;)

Reply:
Fixed 😉

Reply:
@MartinZikmund that's weird, I still see indentation/formating issue on github. Same on your side ?

Reply:
Weird. Looks fine in VS, I did Format Document in VS, so it should be according to editorconfig

Reply:
@MartinZikmund Can you try to check if you have the correct formatting option for XAML please ? (Tools -> Options -> Text Editor -> XAML -> Formatting -> Spacing and on Attribute Spacing section click "Position each attribute on a separate line" and then try "Ctrl+K+D" again.)

Reply:
Ah, I see what you mean now, better :-) ?

Reply:
Yes night and day ;) thanks @MartinZikmund for taking the time!
Changed to resolved !